styles/styles.css

```diff
@@ -0,0 +1,38 @@
+nav{
+    background-color: steelblue;
+    padding: 10px;
+}
+
+nav a {
+    color:white;
+    display: block;
+    padding: 20px;
+}
+
+h1{
+    color: steelblue;
+}
+
+img{
+    width: 150px;
+    height: auto;
+    border: 1px solid steelblue;
+    padding: 2px;
+    box-shadow: 0 0 30px gray;
+} 
+p  { 
+    padding: 10px;
+}
+
+main{
+    max-width: 840px;
+}
+aside{
+    width: 12rem;
+    margin: 2rem;
+    border: 1px solid rgba (0,0,0, 0.2);
+    padding: 1rem;
+    background-color: #eee;
+    color: #333;
+    text-align: center;
+}
```

Comment:
### Error Message
![image](https://github.com/user-attachments/assets/8581c54a-8877-41ec-9260-d0112a0637ed)

### Solution
Refer back to the course content to resolve this issue: [Course Content](https://byui-cse.github.io/wdd130-ww-course/week02/prove-home-enhancement-css.html#:~:text=Set%20the%20img%20inside%20the%20aside%20to%20have%20a%20width%20of%20200%20pixels%20and%20a%20height%20of%20auto).